sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsOperationStatus.cs

```diff
@@ -11,8 +11,7 @@ namespace Azure.AI.TextAnalytics
     [CodeGenModel("State")]
     public partial struct TextAnalyticsOperationStatus
     {
-        /// <summary> Added for compilation. </summary>
-        /// TODO: delete later
-        public static TextAnalyticsOperationStatus Rejected { get; } = new TextAnalyticsOperationStatus(CancellingValue);
+        /// <summary> Rejected </summary>
+        public static TextAnalyticsOperationStatus Rejected { get; } = new TextAnalyticsOperationStatus("Rejected");
```

Comment:
Issue https://github.com/Azure/azure-sdk-for-net/issues/28326
After consideration, decided to add "Rejected" state as this is an extensible enum. This is not sent from the service anymore and after talking to them, they didn't send them before either. 
Even with that, we can't delete it because a customer might have it in their code which will cause a build break.
So, let's leave it as is with the corresponding value